### PR TITLE
Add sample usage

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**

### CODE REVIEW

File: releases/reference-assemblies/README.md [hunk 1]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+    <add key="roslyn-tools" value="https://dotnet.myget.org/f/roslyn-tools/api/v3/index.json" />
+</packageSources>
+</configuration>
+```
+
+Make the following changes based on the type of project you are using.
+
+### SDK-style project
+
+Add `TargetFramework` property and a `PackageReference` like in the following example `csproj` file:
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]:
```suggestion
Add a `TargetFramework` property and a `PackageReference` like in the following example *csproj* file:
```

File: releases/reference-assemblies/README.md [hunk 2]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+  </ItemGroup>
+
+</Project>
+```
+
+### Classic-style project
+
+Add `PackageReference` to your `csproj` file, the same way as in the SDK-style sample above.
+
+Add, or modify, `TargetFrameworkVersion` property in your `csproj` file:
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]:
```suggestion
Add or modify the `TargetFrameworkVersion` property in your *csproj* file:
```

File: releases/reference-assemblies/README.md [hunk 3]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" />
+  </ItemGroup>
+
+</Project>
+```
+
+### Classic-style project
+
+Add `PackageReference` to your `csproj` file, the same way as in the SDK-style sample above.
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]:
```suggestion
1. Add `PackageReference` to your *csproj* file, the same way as in the SDK-style sample above.
```

File: releases/reference-assemblies/README.md (hunk 2 above)
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]:
```suggestion
2. Add, or modify, `TargetFrameworkVersion` property in your `csproj` file:
```

File: releases/reference-assemblies/README.md [hunk 4]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+
+Add, or modify, `TargetFrameworkVersion` property in your `csproj` file:
+
+```xml
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]: This fenced block should be intended 3 spaces.

File: releases/reference-assemblies/README.md [hunk 5]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+<TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+```
+
+### Building your project
+
+1. Restore NuGet packages:
+
+```xml
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]: This fenced block should be indented 3 spaces.

File: releases/reference-assemblies/README.md [hunk 6]
```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
...
+msbuild /t:restore
+```
+
+2. Build the project:
+
+```xml
```
Comment (rpetrusha) [inline releases/reference-assemblies/README.md]: This fenced block should be indented 3 spaces.

File: releases/reference-assemblies/README.md (hunk 1 above)
Comment (NikolaMilosavljevic) [reply releases/reference-assemblies/README.md]: There can only be one TargetFramework property but many PackageReference entries, so I think we should either keep it as is or use something like 'Add the TargetFramework...'. Do you have a current example of this property in docs?

File: releases/reference-assemblies/README.md (hunk 4 above)
Comment (NikolaMilosavljevic) [reply releases/reference-assemblies/README.md]: I've used the indentation model from several existing pages, to keep it consistent, i.e. https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-target-framework-and-target-platform?view=vs-2017, https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x, https://docs.microsoft.com/en-us/dotnet/standard/frameworks

File: releases/reference-assemblies/README.md (hunk 2 above)
Comment (NikolaMilosavljevic) [reply releases/reference-assemblies/README.md]: I've updated the PR.

**Comment by rpetrusha** [comment]
Is your PR ready to merge, @NikolaMilosavljevic?

File: releases/reference-assemblies/README.md (hunk 1 above)
Comment (rpetrusha) [reply releases/reference-assemblies/README.md]: It's prefaced either with "a" or "the" in the doc set. In any case, there needs to be an article before TargetFramework. Either "the" or "a" is fine.

Comment (NikolaMilosavljevic) [reply releases/reference-assemblies/README.md]: I've updated the PR.

**Pull request merged**